examples/monolithic_build/mlkem_native_monobuild.c

```diff
@@ -69,13 +71,13 @@
 #include "mlkem/sampling.c"
 #include "mlkem/verify.c"
 
-#if !defined(MLK_CONFIG_MONOBUILD_CUSTOM_FIPS202)
+#if !defined(MLK_CONFIG_FIPS202_CUSTOM_HEADER)
```

Comment:
can we somewhere add an error if `MLK_CONFIG_FIPS202_CUSTOM_HEADER` is set,  but `MLK_CONFIG_FIPS202X4_CUSTOM_HEADER` isn't? I ran into this when doing the pqm4 integration.

Reply:
Yes, but let's to that in a follow-up.